com.unity.shadergraph/CHANGELOG.md

```diff
@@ -8,11 +8,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
   - Stereo Eye Index, Instance ID, and Vertex ID nodes added to the shadergraph library.
+  - Added information about selecting and unselecting items to the Blackboard article.
 
 ### Changed
 - Change Asset/Create/Shader/Blank Shader Graph to Asset/Create/Shader Graph/Blank Shader Graph
 - Change Asset/Create/Shader/Sub Graph to Asset/Create/Shader Graph/Sub Graph
 - Change Asset/Create/Shader/VFX Shader Graph to Asset/Create/Shader Graph/VFX Shader Graph
+- Adjusted Blackboard article to solve DOC-2592
```

Comment:
The Jira ticket number is not useful in the changelog, please describe the actual issue you're solving, and optionally add a link to a public IssueTracker card (if there is one).

Reply:
oops, had fixed that on the last PR of this branch and then somehow it got unfixed, sorry.